src/api.py

```diff
@@ -77,26 +77,38 @@ def answer(self, question: str, chat_session_id: Optional[str] = None) -> Dict[s
     
 
     @post("respond", public=True)
-    def respond(self, update_id: int, message: dict) -> InvocableResponse[str]:
+    def respond(self, update_id: int, message: dict, **kwargs) -> InvocableResponse[str]:
         """Endpoint implementing the Telegram WebHook contract. This is a PUBLIC endpoint since Telegram cannot pass a Bearer token."""
-        message_text = message['text']
-        chat_id = message['chat']['id']
-        message_id = message['message_id']
 
-        # TODO: must reject things not from the package
-        try:
+        message_text = (message or {}).get('text', "")
+        if message_text is not None and message_text != "":
```

Comment:
my only nit (NON-BLOCKING):

i'd write this with the default cases up top, a la:

```
if not message_text:
  return InvocableResponse(string="OK")

if len(message_text) == 0:
  return InvocableResponse(string="OK")

...
```

Reply:
Done.